pkg/wsman/cim/models/constants.go

```diff
@@ -233,201 +403,201 @@ const (
-	UpgradeMethodSocketFM1
-	UpgradeMethodSocketFM2
-	UpgradeMethodSocketLGA20113
-	UpgradeMethodSocketLGA13563
-	UpgradeMethodSocketLGA1150
-	UpgradeMethodSocketBGA1168
-	UpgradeMethodSocketBGA1234
-	UpgradeMethodSocketBGA1364
-	UpgradeMethodSocketAM4
-	UpgradeMethodSocketLGA1151
-	UpgradeMethodSocketBGA1356
-	UpgradeMethodSocketBGA1440
-	UpgradeMethodSocketBGA1515
-	UpgradeMethodSocketLGA36471
-	UpgradeMethodSocketSP3
-	UpgradeMethodSocketSP3r2
-	UpgradeMethodSocketLGA2066
-	UpgradeMethodSocketBGA1392
-	UpgradeMethodSocketBGA1510
-	UpgradeMethodSocketBGA1528
-	UpgradeMethodSocketLGA4189
-	UpgradeMethodSocketLGA1200
-	UpgradeMethodSocketLGA4677
-	UpgradeMethodSocketLGA1700
-	UpgradeMethodSocketBGA1744
-	UpgradeMethodSocketBGA1781
-	UpgradeMethodSocketBGA1211
-	UpgradeMethodSocketBGA2422
-	UpgradeMethodSocketLGA1211
-	UpgradeMethodSocketLGA2422
-	UpgradeMethodSocketLGA5773
-	UpgradeMethodSocketBGA5773
-)
-const (
-	CPUStatusUnknown CPUStatus = iota
-	CPUStatusEnabled
-	CPUStatusDisabledByUser
-	CPUStatusDisabledByBIOS
-	CPUStatusIdle
-	CPUStatusOther
+	UpgradeMethod_Other UpgradeMethod = iota + 1
+	UpgradeMethod_Unknown
+	UpgradeMethod_DaughterBoard
+	UpgradeMethod_ZIFSocket
+	UpgradeMethod_ReplacementPiggyBack
+	UpgradeMethod_None
+	UpgradeMethod_LIFSocket
+	UpgradeMethod_Slot1
+	UpgradeMethod_Slot2
+	UpgradeMethod_370PinSocket
+	UpgradeMethod_SlotA
+	UpgradeMethod_SlotM
+	UpgradeMethod_Socket423
+	UpgradeMethod_SocketA
+	UpgradeMethod_Socket478
+	UpgradeMethod_Socket754
+	UpgradeMethod_Socket940
+	UpgradeMethod_Socket939
+	UpgradeMethod_SocketmPGA604
+	UpgradeMethod_SocketLGA771
+	UpgradeMethod_SocketLGA775
+	UpgradeMethod_SocketS1
+	UpgradeMethod_SocketAM2
+	UpgradeMethod_SocketF1207
+	UpgradeMethod_SocketLGA1366
+	UpgradeMethod_SocketG34
+	UpgradeMethod_SocketAM3
+	UpgradeMethod_SocketC32
+	UpgradeMethod_SocketLGA1156
+	UpgradeMethod_SocketLGA1567
+	UpgradeMethod_SocketPGA988A
+	UpgradeMethod_SocketBGA1288
+	UpgradeMethod_rPGA988B
+	UpgradeMethod_BGA1023
+	UpgradeMethod_BGA1224
+	UpgradeMethod_LGA1155
+	UpgradeMethod_LGA1356
+	UpgradeMethod_LGA2011
+	UpgradeMethod_SocketFS1
+	UpgradeMethod_SocketFS2
+	UpgradeMethod_SocketFM1
+	UpgradeMethod_SocketFM2
+	UpgradeMethod_SocketLGA20113
+	UpgradeMethod_SocketLGA13563
+	UpgradeMethod_SocketLGA1150
+	UpgradeMethod_SocketBGA1168
+	UpgradeMethod_SocketBGA1234
+	UpgradeMethod_SocketBGA1364
+	UpgradeMethod_SocketAM4
+	UpgradeMethod_SocketLGA1151
+	UpgradeMethod_SocketBGA1356
+	UpgradeMethod_SocketBGA1440
+	UpgradeMethod_SocketBGA1515
+	UpgradeMethod_SocketLGA36471
+	UpgradeMethod_SocketSP3
+	UpgradeMethod_SocketSP3r2
+	UpgradeMethod_SocketLGA2066
+	UpgradeMethod_SocketBGA1392
+	UpgradeMethod_SocketBGA1510
+	UpgradeMethod_SocketBGA1528
+	UpgradeMethod_SocketLGA4189
+	UpgradeMethod_SocketLGA1200
+	UpgradeMethod_SocketLGA4677
+	UpgradeMethod_SocketLGA1700
+	UpgradeMethod_SocketBGA1744
+	UpgradeMethod_SocketBGA1781
+	UpgradeMethod_SocketBGA1211
+	UpgradeMethod_SocketBGA2422
+	UpgradeMethod_SocketLGA1211
```

Comment:
line 520 should be LGA5773 according to UpgradeMethod under CIM_processor.  Need  to confirm order/values after this line as well.  Everything  before this line looked good..

Reply:
fixed

Reply:
Fixed, however, it shouldn't be used.  Also, AMT SDK seems to be missing 2 items? (85 of 87 string values in the value map)